flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -356,7 +356,9 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
     if (args.isNotEmpty()) {
       val event = args.first() as ByteArray?
       if (event != null && event.isNotEmpty()) {
-        val id = InternalSentrySdk.captureEnvelope(event)
+        // Flutter event do not crash the application, we always want a new session to start
+        // after event with exception.handled = false
+        val id = InternalSentrySdk.captureEnvelope(event, true)
```

Comment:
I'm not sure if always setting it to true will give us the right behaviour

I think we need to check first if it is handled since `captureEnvelope` is also used in handled exceptions when a user does `captureException` manually